packages/manager/src/features/OneClickApps/oneClickApps.ts

```diff
@@ -1122,6 +1122,116 @@ export const oneClickApps: OCA[] = [
       'Powerful anti-spam and email security solution for control panels (including cPanel and Plesk).',
     website: 'https://www.magicspam.com/anti-spam-features.php',
   },
+  {
+    alt_description: 'FFmpeg encoder plugins.',
+    alt_name: 'Premium video encoding',
+    categories: ['Media and Entertainment'],
+    colors: {
+      end: '041125',
+      start: '6DBA98',
+    },
+    description: `MainConcept FFmpeg Plugins are suited for both VOD and live production workflows, with advanced features such as Hybrid GPU acceleration and xHE-AAC audio format.`,
+    logo_url: 'mcffmpegplugin.svg',
```

Comment:
Should this be
```suggestion
    logo_url: 'mcffmpegplugins.svg',
```
Or should the .svg filename be changed?